OpenRA.Game/Map/MapGrid.cs

```diff
@@ -13,11 +13,11 @@
 
 namespace OpenRA
 {
-	public enum TileShape { Rectangle, Diamond }
+	public enum MapGridType { Rectangular, RectangularIsometric }
```

Comment:
I suspect that there is a plan to include `DiamondIsometric` which would be for diamond shaped isometric maps instead of rectangular (isometric) maps.